apps/aecore/lib/aecore/chain/worker.ex

```diff
@@ -189,6 +200,32 @@ defmodule Aecore.Chain.Worker do
     {:reply, txs_index, state}
   end
 
+  def handle_info(:timeout, state) do
```

Comment:
so this will timeout with a bigger blockchain, as everything is loaded in memory, but we can keep it for now and improve with a later task